pkg/signature/manifest.go

```diff
@@ -32,8 +32,16 @@ type GojqParams struct {
 }
 
 var (
-	AlwaysExcluded = []string{".status", ".metadata.creationTimestamp", ".metadata.generation", ".metadata.managedFields", ".metadata.uid", ".metadata.resourceVersion"}
-	GojqTemplate   = template.Must(template.New("gojq_query").Funcs(
+	AlwaysExcluded = []string{
+		".status",
+		".metadata.creationTimestamp",
+		".metadata.generation",
+		".metadata.managedFields",
+		".metadata.uid",
+		".metadata.resourceVersion",
```

Comment:
what are your thoughts on adding `.metadata.name`? It would be useful to be able to duplicate a bundle.

Reply:
I actually commented a couple days back on the issue you created for this: https://github.com/aws/eks-anywhere-packages/issues/598

I think it would allow to fake upgrades which are effectively downgrades that may have CVEs. We don't really support downgrading either since we don't have downwards migrations so this could potentially make it easy for attackers to break things.